03_building_and_packaging/spack_slides.md

```diff
@@ -38,9 +38,9 @@ slideOptions:
 
 ## Learning goals
 
-- What are the challenges when bringing (your) software to supercomputers?
-- How to use Spack to install software.
-- How to create a Spack package for your own software.
+- Understand the challenges when bringing (your) software to supercomputers.
+- Use Spack to install software.
+- Create a Spack package for your own software.
```

Comment:
@uekerman learning goals okay?

Reply:
Better be more specific here. "Understand" is in general discouraged as too unspecific. Also in other two points, you could be more specific. Should students be able to create Spack packages for any software?

Reply:
> Should students be able to create Spack packages for any software?

In principle yes. We show them how to create the package in a reasonably generic way.

I will be more specific with the learning goals and directly update on the main branch.